sdk/textanalytics/azure-ai-textanalytics/src/samples/java/com/azure/ai/textanalytics/TextAnalyticsAsyncClientJavaDocCodeSnippets.java

```diff
@@ -784,4 +791,82 @@ public void analyzeBatchSentimentMaxOverloadWithOpinionMining() {
             });
         // END: com.azure.ai.textanalytics.TextAnalyticsAsyncClient.analyzeSentimentBatch#Iterable-AnalyzeSentimentOptions
     }
+
+    // Healthcare
+    /**
+     * Code snippet for {@link TextAnalyticsAsyncClient#beginAnalyzeHealthcare(Iterable, RecognizeHealthcareEntityOptions)}
+     */
+    public void analyzeBatchHealthcareMaxOverloadWithOpinionMining() {
```

Comment:
It took me a while to understand what this was haha I think you need to update the method name :P

Reply:
Ahh. Sorry for that. Trying to follow the pattern of Sentiment and other batch documents analysis endpoints.